modules/post/windows/gather/forensics/fanny_bmp_check.rb

```diff
@@ -0,0 +1,63 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+
+require 'msf/core/post/common'
+require 'msf/core/post/windows/priv'
+class MetasploitModule < Msf::Post
+
+        include Msf::Post::Common
+        include Msf::Post::Windows::Registry
+
+        def initialize(info={})
+                super( update_info( info,
+                                 'Name' => 'FannyBMP or Dementiawheel Detection Registry Check',
+                                 'Description' => %q{This module searches for the Fanny.bmp worm related registry keys.
+                                 fanny.bmp is a worm that exploited zero day vulns
+                                 (more specifically, the LNK Exploit CVE-2010-2568).
+                                 Which allowed it to spread even if USB Autorun was turned off.
+                                 This is exactly the same Exploit that was used in StuxNet.},
+                                 'License' => MSF_LICENSE,
+                                 'Author' => [ 'William M.'],
+                                 'Platform' => [ 'win' ],
+                                 'SessionTypes' => [ 'meterpreter','shell'],
+                                 'References' => [[ 'URL', 'https://securelist.com/a-fanny-equation-i-am-your-father-stuxnet/68787']
+                                 ,['CVE', '2010-2568']]
+                        ))
+
+
+        end
+        def run
+
+                                query = 'HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\MediaResources\acm\"ECELP4",'
+                                query += 'HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\MediaResources\acm\ECELP4\Driver,'
+                                query += 'HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\MediaResources\acm\ECELP4\filter2,'
+                                query += 'HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\MediaResources\acm\ECELP4\filter3,'
+                                query += 'HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\MediaResources\acm\ECELP4\filter8'
+                match = 0
+                                print_status("Searching registry on Target for Fanny.bmp artifacts.")
+                keys = query.split(/,/)
```

Comment:
Please adjust the whitespace in the entire module, the metasploit codebase standard is `2 spaces` per indentation level when required.  Consider this entire block from line 33 to 40 each line should be at the same level.

Reply:
Of course,  Copy that!
Thanks!